apps/meteor/server/methods/reportMessage.ts

```diff
@@ -43,13 +54,30 @@ Meteor.methods<ServerMethods>({
 		const { rid } = message;
 		// If the user can't access the room where the message is, report that the message id is invalid
 		const room = await Rooms.findOneById(rid);
-		if (!room || !(await canAccessRoomAsync(room, { _id: uid }))) {
+		if (!room || !(await canAccessRoomAsync(room, { _id: user._id }))) {
 			throw new Meteor.Error('error-invalid-message_id', 'Invalid message id', {
 				method: 'reportMessage',
 			});
 		}
 
-		await Reports.createWithMessageDescriptionAndUserId(message, description, uid);
+		const reportedBy = {
+			_id: user._id,
+			username: user.username,
+			name: user.name,
+			active: user.active,
+			avatarETag: user.avatarETag,
```

Comment:
please remove `avatarETag` storing this value makes no sense

Reply:
Thank you! I have removed it.